README.md

```diff
@@ -36,26 +36,44 @@ Features:
 * Supports ensembles of multiple data streams, and supports common methods for temporal alignment of streams.
 
 ## Installation
-To install the latest stable release of **neuroconv** though PyPI, type:
+We always recommend installing and running Python packages in a clean environment. One way to do this is via [conda environments](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment):
+
+```shell
+conda create --name <give the environment a name> --python <choose a version of Python to use>
+conda activate <environment name>
+```
+
+To install the latest stable release of **neuroconv** though PyPI, run:
+
 ```shell
 pip install neuroconv
 ```
 
-For more flexibility we recommend installing the latest version directly from GitHub. The following commands create an environment with all the required dependencies and the latest updates:
+To install the current unreleased `main` branch (requires `git` to be installed in your environment, such was via `conda install git`), run:
+
+```shell
+pip install git+https://github.com/catalystneuro/neuroconv.git@main
+```
+
+To install the package in [editable mode](https://pip.pypa.io/en/stable/cli/pip_install/#editable-installs), run:
```

Comment:
Just for your information.

For another PR as you are just refactoring but I think we probably should remove mentions of editable installs on general readme documents and send them over to the developer sections. 

With the migration away from setup tools editable installs are less and less consistent I jumped into this rabbit hole last week. Just two point out two common problems:

* Poetry which is one of the most pouplar npm-like experiences for python does not suppport editable installs.
* static analysis tools would not be able to use the enabled editable installs because they use import hooks:

https://github.com/microsoft/pylance-release/issues/3473

I can take care of this later.

Reply:
Removed editable install instructions; I believe the idea of this README duplicating our user guide in the docs is to reduce redirection for people just finding the package on PyPI or main GitHub page. So just quickest and dirtiest instructions to get going. Otherwise I'm in greater favor of going the GUIDE README approach and offloading everything of detail to the main docs

Reply:
Good. Not sure if they should be removed from the cookie cutter for the conversion projects as well. Here we expect users, there we expect people to change stuff? leaning to remove it as well.

Reply:
Conversion projects depend on the client. Some want/need to tamper with stuff, so yes to editable there

Others want static PyPI release with CLI tooling; what we do most with them with respect to bug fixes is do the `git` based `pip` install of a particular branch or commit